massa-client/src/tests/mod.rs

```diff
@@ -1,3 +1,3 @@
-#[cfg(target_feature = "local")]
```

Comment:
In fact the test was already disabled but in a way a bit non-obvious (and accidental?) since feature `local` didn't exist...

Reply:
That's weird

Reply:
I did that, it wasn't documented, sorry :slightly_smiling_face: because one day we said that we didn't want to test that on the CI but only if we specify it in local.

Reply:
Indeed x)